src/ZfcUser/Controller/UserController.php

```diff
@@ -111,7 +110,13 @@ public function loginAction()
         $form->setData($request->getPost());
 
         if (!$form->isValid()) {
-            $this->flashMessenger()->setNamespace('zfcuser-login-form')->addMessage($this->failedLoginMessage);
+            if ($this->getServiceLocator()->has('translator')) {
```

Comment:
Translater should be injected. It's not good to have a dependency on service-locator